File: online-judge-solutions/Codeforces/474A.cpp
//https:codeforces.com/problemset/problem/474/A/
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	100 KB
#include<iostream>
#include<map>
using namespace std;
int main(){
	char c;cin>>c;
	string s,r;cin>>s;
	string k="qwertyuiopasdfghjkl;zxcvbnm,./";
	map<char,int> m;
	for(int i=0;i<k.size();i++) m[k[i]]=i;
	for(int x:s) cout<<(c=='R'?k[m[x]-1]:k[m[x]+1]);cout<<endl;
	return 0;
}

File: online-judge-solutions/Codeforces/474A.py
#https://codeforces.com/problemset/problem/474/A/
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
c,s,k,m=input(),input(),"qwertyuiopasdfghjkl;zxcvbnm,./",{}
m={z:i for i,z in enumerate(k)}
print(*[k[m[x]-1] if c=='R' else k[m[x]+1] for x in s],sep="")
